travis.php.ini

```diff
@@ -0,0 +1 @@
+memory_limit = 2G
```

Comment:
Is this really needed?

If so, we must add it to `.gitattributes` and also remove the `minimum-stability` from `composer.json`